src/main/java/baritone/process/MineProcess.java

```diff
@@ -409,6 +410,8 @@ private void addNearby() {
                 // remove any that are implausible to mine (encased in bedrock, or touching lava)
                 .filter(pos -> MineProcess.plausibleToBreak(ctx, pos))
 
+                .filter(pos -> isNextToAir(ctx, pos))
```

Comment:
Shouldn't this only calculate when the setting is enabled?

Reply:
ya